src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -2053,6 +2053,30 @@ public void PreviousSubmissionWithError()
             Assert.Throws<InvalidOperationException>(() => CreateSubmission("a + 1", previous: s0));
         }
 
+        [Fact]
+        public void CreateTupleTypeSymbol()
```

Comment:
Have we annotated this **class** with `[CompilerTrait(CompilerFeature.Tuples)]`? If not we should to facilitate tracking these tests.  I'm really curious how many we have for tuples.  Interested in comparing it to say local functions.